Src/Orbiter/FlightRecorder.cpp

```diff
@@ -96,7 +96,7 @@ void Vessel::FRecorder_Activate (bool active, const char *fname, bool append)
 		if (!append) FRecorder_Reset();
 		bFRrecord = true;
 		char cbuf[256];
-		sprintf (cbuf, "Flights\\%s\\%s.pos", fname, name);
+		sprintf (cbuf, "Flights\\%s\\%s.pos", fname, name.c_str());
```

Comment:
If we ever want to make Orbiter cross-platform, which we do, all paths should be converted to use forward slash.

If you look under [Path normalization](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#path-normalization) section, it says:

> ## Canonicalize separators
> All forward slashes (`/`) are converted into the standard Windows separator, the back slash (`\`). If they are present, a series of slashes that follow the first two slashes are collapsed into a single slash.

Reply:
Plus, this line should really be something like:
```c++
auto path = fs::path{"Flights"} / fname / name + ".pos";
```
and not a stupid `sprintf`.

NB: For this to work `name` would need to be am `std::string` and not `char*`.